MigrationTutorial/Models/V2/Department.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.Linq;
+using MongoDB.Bson;
+using Realms;
+
+namespace MigrationTutorial.Models.V2
+{
+    public class Department : RealmObject
+    {
+        [PrimaryKey]
+        public ObjectId Id { get; private set; } = ObjectId.GenerateNewId();
+
+        [Required]
+        public string? Name { get; set; }
+
+        //[Required]
```

Comment:
I suppose this is here for testing?

Reply:
I left it for myself as a note to remember to ask: what's the recommended way to simulate a requirement on non-supported fields?

Reply:
Aaah, that's a though one, you're practically talking about validation and... I don't think there is an easy way to do it here. I also think you should not worry too much about this, otherwise it becomes an article about modeling 😁